src/seeders/20230317073843-dummy-user.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable linebreak-style */
+/**
+ * @param {data} queryInterface .
+ * @returns {data} .
+ */
+export async function up(queryInterface) {
+  return queryInterface.bulkInsert('Dummy', [{
+    names: 'Dummy User',
+    email: 'dummy@email.com',
+    password: 'dummypassword',
```

Comment:
this password needs to be hashed otherwise when logging in this dummy user the right way, it will fail

Reply:
corrected